### PR TITLE
Fix auth bugs with concepts. Pip install lilac[all] in the dockerfile.

### PR DESCRIPTION
**Pull request by nsthorat**
- Fix auth bugs with adding concepts.
- Make sure to install lilac[all] in the huggingface space so we can run everything.

Compare: https://lilacai-lilac.hf.space/datasets#lilac/mmlu_professional_law
to:
https://lilacai-nikhil-staging.hf.space/datasets#lilac/mmlu_professional_law

Towards https://github.com/lilacai/lilac/issues/601

### CODE REVIEW

**Review by dsmilkov — APPROVED** [review]

File: web/lib/src/lilac.ts [hunk 1]
```diff
@@ -38,7 +38,6 @@ export type Filter = BinaryFilter | UnaryFilter | ListFilter;
 
 export type LilacField<S extends Signal = Signal> = Field & {
   path: Path;
-  parent?: LilacField;
```
[the remark this answers is not in the document]
Comment (dsmilkov) [reply web/lib/src/lilac.ts:41]: oh i misunderstood, the problem is that field is no longer json serializable so tanstack can't use it as cache key, got it

[the remark this answers is not in the document]
Comment (dsmilkov) [reply web/lib/src/lilac.ts:41]: Found the root cause and sent you a 1-liner in https://github.com/lilacai/lilac/pull/625

File: web/blueprint/src/lib/queries/datasetQueries.ts [hunk 2]
```diff
@@ -121,7 +121,7 @@ export const querySelectRows = createApiQuery(async function selectRows(
     total_num_rows: res.total_num_rows
   };
 },
-DATASETS_TAG);
+'DATASETS_TAG');
```
Comment (dsmilkov) [inline web/blueprint/src/lib/queries/datasetQueries.ts:124]: this looks strange

Comment (nsthorat) [reply web/blueprint/src/lib/queries/datasetQueries.ts]: done

**Pull request merged**